static/src/stylesheets/module/commercial/gustyle/_gu-comlabel.scss

```diff
@@ -0,0 +1,33 @@
+.gu-comlabel__btn {
+    @include f-textSans;
+    @include font-size(14, 18);
+
```

Comment:
Yes, to me it makes sense to group includes and extends together and then other styles. But it is very subjective.